app/src/main/java/com/devindi/wallpaper/misc/SimpleSeekBarChangeListener.kt

```diff
@@ -0,0 +1,28 @@
+package com.devindi.wallpaper.misc
+
+import android.widget.SeekBar
+
+/**
+ * Simple [SeekBar.OnSeekBarChangeListener] implementation with progress observer only.
+ *
+ * Usage sample:
+ * seekBar.setOnSeekBarChangeListener(SimpleSeekBarChangeListener { value, fromUser ->
+ *   if (fromUser) {
+ *       updateValue(value)
+ *   }
+ * })
+ *
+ */
+class SimpleSeekBarChangeListener(private val observer: (progress: Int, fromUser: Boolean) -> Unit) : SeekBar.OnSeekBarChangeListener {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/f22b9ee68b14344acb1bd6cd546c2767866b4eef/app/src/main/java/com/devindi/wallpaper/misc/SimpleSeekBarChangeListener.kt#L16"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>